students/MikeSchincariol/session6/scoring.py

```diff
@@ -0,0 +1,17 @@
+def hand_weapon():
+    return {'small': 1,
+            'medium': 2,
+            'large': 3}
+
+def gun():
+    return {'small': 5,
+            'medium': 8,
+            'large': 13}
+
+def flower_power():
+    return {'small': 21,
+            'medium': 34,
+            'large': 55}
+
+def score(weapon_type, weapon_size):
```

Comment:
Nice and clean here too.  I used a dictionary as switch approach in my solution for readability whereas you went straight to returning the dictionary.  I like both approaches.